spec.md

```diff
@@ -2153,6 +2158,46 @@ operand (when the stride is 1). By contrast, slicing a list requires
 the creation of a new list and copying of the necessary elements.
 
 
+### Lambda expressions
+
+A `lambda` expression yields a new function value.
+
+```grammar {.good}
+LambdaExpr = 'lambda' [Parameters] ':' Test .
+```
+
+Syntactically, a lambda expression consists of the keyword `lambda`,
+followed by a parameter list like that of a `def` statement but
+unparenthesized, then a colon `:`, and a single expression, the
+_function body_.
+
+Example:
+
+```python
+def map(f, list):
+    return [f(x) for x in list]
+
+map(lambda x: 2*x, range(3))    # [2, 4, 6]
+```
+
+As with functions created by a `def` statement, a lambda function
+captures the syntax of its body, the default values of any optional
+parameters, the value of each free variable appearing in its body, and
```

Comment:
> value of each free variable

"Value" is wrong, it's the cell (or whatever the user-facing terminology for this concept would be).

Reply:
Changed to "reference to each free variable appearing in the body". Added an example.